index.html

```diff
@@ -1,12 +1,77 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Weather Report</title>
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Cute+Font&family=Righteous&display=swap');
+    </style>
 </head>
-<body>
+    <body>
+    <header>
+    <link rel="stylesheet" href="styles/index.css" />
+        <h1>Weather</h1>
+        <h1>Report</h1>
+        <marquee><h3>********************************************
+              Weather Report 24/7 ********************************************
+              Weather Report 24/7 ******************************************** </h3></marquee>
```

Comment:
Your heading tags should be able to form a table of contents and shouldn't be used just for styling (you should use CSS classes to resize things instead).

Also, the `<marquee>` tag is deprecated and shouldn't be used anymore: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/marquee